lib/matplotlib/axes/_axes.py

```diff
@@ -452,8 +452,10 @@ def indicate_inset(self, bounds, inset_ax=None, *, transform=None,
         edgecolor : :mpltype:`color`, default: '0.5'
             Color of the rectangle and color of the connecting lines.
 
-        alpha : float, default: 0.5
-            Transparency of the rectangle and connector lines.
+        alpha : float or None, default: 0.5
```

Comment:
Agree that this is not ideal, because you cannot set face and edge transparencies independently without first resetting alpha.

In general, this seems not too useful by default: I would have expected the edgecolors to be solid (=frame should be above the plot). There is no facecolor by default. The only thing this serves is that you can get a semi-transparent face by specifying a simple solid color - OTOH that's confusing.

I suggest to change this to `None` by default. Optionally, change edgecolor to be the solid-equivalent color of `('0.5', 0.5)` against a white background. But I'm inclined to see both the grey "0.5" and the alpha=0.5 as uncoordinated ways to lighten the frame and suspect using both is not intentional. So possibly, just keep edgecolor as is.

Overall I'd just do it and call it a bug fix.

Reply:
alpha = 0.5 was originally for the lines that indicate the inset which often cross data, and hence its nice to have some indication of the data underneath.  It was not meant to apply to the facecolor of the inset indicator patch.

Reply:
The current defaults make sense to me.  The point of these patches is to point to some other features we are _indicating_, so they should interfere as little as possible with our view of those things.  I think most of the time a user would not want a facecolor because that would make it harder to see the colours of the thing they are _indicating_.  With this PR we will have documented how to change the alpha for the rarer occasions when a facecolor is wanted and the face/edge alpha should be different.